Data/ByteString/Short/Internal.hs

```diff
@@ -336,20 +300,15 @@ instance Read ShortByteString where
 -- | @since 0.10.12.0
 instance GHC.Exts.IsList ShortByteString where
   type Item ShortByteString = Word8
-  fromList = packBytes
-  toList   = unpack
+  fromList  = ShortByteString . GHC.Exts.fromList
+  fromListN = (ShortByteString .) . GHC.Exts.fromListN
```

Comment:
Does this implementation of `fromListN` behave differently than the old default implementation?

Reply:
At the moment they are the same.

Reply:
Nice catch. They are _not_ the same. Looking at `Data.Array.Byte`, this implementation is faster but will error out if the length parameter is wrong. Perhaps it's worth a changelog entry.